website/docs/r/codestarconnections_connection.markdown

```diff
@@ -68,6 +68,7 @@ The following arguments are supported:
 
 * `name` - (Required) The name of the connection to be created. The name must be unique in the calling AWS account. Changing `name` will create a new resource.
 * `provider_type` - (Required) The name of the external provider where your third-party code repository is configured. Valid values are `Bitbucket`, `GitHub`, or `GitHubEnterpriseServer`. Changing `provider_type` will create a new resource.
+* `tags` - (Optional) An array of key:value pairs to associate with the resource.
```

Comment:
In Terraform the type is [`map`](https://www.terraform.io/docs/language/expressions/types.html)

```suggestion
* `tags` - (Optional) Map of key-value resource tags to associate with the resource.
```